### PR TITLE
[BUGFIX] Fix phpcs fix script

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #246

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
 			"@fix:php:sniff"
 		],
 		"fix:php:cs": "./tools/php-cs-fixer fix --config .php_cs.php",
-		"fix:php:sniff": ".Build/vendor/bin/phpcs Classes Configuration Tests",
+		"fix:php:sniff": ".Build/vendor/bin/phpcbf Classes Configuration Tests",
```
Comment (oliverklee) [inline composer.json:119]: Please let's add `phpcbf` to the PHIVE-installed tools and use that.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (oliverklee) [reply composer.json:119]: Also, we currently do not have PHP_CodeSniffer installed as a Composer dependency.

Comment (lukaszuznanski) [reply composer.json:119]: alright, that's true :)

File: composer.json [hunk 2]
```diff
@@ -117,7 +117,7 @@
...
```
Comment (oliverklee) [inline composer.json:120]: We'll need to refer to the PHAR here, not the Composer-installed executable.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — APPROVED** [review]

**Pull request merged**